### PR TITLE
Return Objects instead of string values for Swipe and Pinch

### PR DESCRIPTION
**Pull request by luiscarlosjayk**
...th left,right and in, out boolean values, respectively.

For Swipe.js
event.direction.left and event.direction.right are boolean values
For Pinch.js
event.direction.in and event.direction.out are boolean values.
I find this way more comfortable than comparing to strings.

### CODE REVIEW

File: Source/Touch/Pinch.js [hunk 1]
```diff
@@ -1,66 +1,73 @@
...
+		if (disabled || !active) return;
+
+		event.preventDefault();
+
+		var threshold = this.retrieve(thresholdKey, 0.5);
+		if (event.scale < (1 + threshold) && event.scale > (1 - threshold)) return;
+
+		active = false;
+		event.pinch = (event.scale > 1) ? 'in' : 'out';
+		event.direction = {};
```
Comment (luiscarlosjayk) [inline Source/Touch/Pinch.js:41]: starting from here

File: Source/Touch/Pinch.js [hunk 2]
```diff
@@ -1,66 +1,73 @@
...
+		event.direction.in = event.pinch == 'in';
+		event.direction.out = event.pinch == 'out';
+		event.direction.toString = function() {
+			return	event.pinch;
+		}
```
Comment (luiscarlosjayk) [inline Source/Touch/Pinch.js:46]: up to here, and that's it for Pinch.js

File: Source/Touch/Swipe.js [hunk 3]
```diff
@@ -1,95 +1,102 @@
...
+			delta = end.x - start.x,
+			isLeftSwipe = delta < -distance,
+			isRightSwipe = delta > distance;
+
+		if (!isRightSwipe && !isLeftSwipe)
+			return;
+		
+		event.preventDefault();
+		active = false;
+		// event.direction = (isLeftSwipe ? 'left' : 'right');
```
Comment (luiscarlosjayk) [inline Source/Touch/Swipe.js:62]: changes, starting from here

File: Source/Touch/Swipe.js [hunk 4]
```diff
@@ -1,95 +1,102 @@
...
+		event.direction = {};
+		event.direction.left = isLeftSwipe;
+		event.direction.right = isRightSwipe;
+		event.direction.toString = function() {
+			var side = (isLeftSwipe ? 'left' : 'right');
+			return	side;
+		}
```
Comment (luiscarlosjayk) [inline Source/Touch/Swipe.js:69]: changes end here for Swipe.js

notice I kept an event.direction.toString method trying to keep compatibility with older string values

**Comment by cpojer** [comment]
Hello,

again, I am not going to pull this without a proper diff.

**Comment by cpojer** [comment]
As I said before, please redo your changes with proper newlines (Linux \n instead of Windows). If you do not know how to do this, I'm sure a google search will help you. Otherwise feel free to join #mootools on Freenode.

**Pull request closed without merging**